src/Platform/Microsoft.Testing.Platform/Requests/RequestNotSupportedException.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Testing.Platform.Requests;
+
+[Serializable]
+public class RequestNotSupportedException : Exception
```

Comment:
Shall we have the usual ctors? Message and inner exception?

Reply:
I was wondering if make sense...the platform doesn't needs any message, I think we will take the exception and we will do something like
* print in console
* crash
* move on if we're good with it...

So I wonder what the test framework author could put in that message.